### DISCUSSION TITLE
SQANTI3 rescue:object 'filter_result' not found

### DISCUSSION THREAD

**Discussion by MR-D-CJ** [ConesaLab/SQANTI3#319] [General] [answered]
The following is the log file of sqanti3_rescue.py:
![image](https://github.com/user-attachments/assets/a9643789-d810-4bf1-b6e3-4649cff37d30)

this is my command：
![image](https://github.com/user-attachments/assets/629d22d0-9d90-48f5-9cf1-d495abc0677b)

**Comment by raysinensis** [comment, marked as answer]
Had the same issue. This error is caused by passing the classification file generated by squanti3_qc instead of a similar file generated by the next step, squanti3_filter.

The file it wants should be named "somethingsomething_RulesFilter_result_classification.txt"
